server/src/main/java/com/paypal/selion/grid/JarSpawner.java

```diff
@@ -45,14 +45,14 @@ public JarSpawner(String[] args, ProcessLauncherOptions options) {
         init(args, options);
     }
 
-    public static final void main(String[] args) {
+    public static void main(String[] args) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1529806151/issues/source?bid=2928425&fileBranchId=2989551#l48)